src/Deserialization.purs

```diff
@@ -0,0 +1,65 @@
+module Deserialization
+  ( convertAddress
+  ) where
+
+import Control.Alt ((<|>))
+import Data.Maybe (Maybe(..))
```

Comment:
Why for `Maybe`?

Also, psc-ide-mode automatically fixes implicit imports by inserting a list of imported things with `(..)` for constructors, so fixing these would be a lot of manual labor for me ^^'

Reply:
Mainly because it's mandated by the **Haskell** style guide: https://github.com/mlabs-haskell/styleguide
I don't really mind if you ignore it as someone will likely adhere to the guide at the end :D I tend to do it when I can so it saves on work later on.

Reply:
No I think that's important. Just forgot it's in the guide.